scripts/rollup/shims/react-native/NativeMethodsMixin.js

```diff
@@ -14,7 +14,7 @@ const {
   __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED,
 } = require('ReactNative');
 
-import type {NativeMethodsMixinType} from 'ReactNativeTypes';
```

Comment:
These (and below) are wrong because they're written to be copied into RN repo.

Reply:
Oh, don't know that, could we add some comments about that, I think that would be better?

Reply:
Done. Remove this and below